test/Table_Tests/src/Common_Table_Operations/Date_Time_Spec.enso

```diff
@@ -84,13 +84,15 @@ add_specs suite_builder setup =
         table_builder cols =
             setup.table_builder cols connection=data.connection
 
-        group_builder.specify "should allow to get the year/month/day of a Date" <|
+        group_builder.specify "should allow to get the year/month/day/day_of_year/day_of_week of a Date" <|
             t = data.dates
             a = t.at "A"
             a.year . to_vector . should_equal [2020, 2024, 1990, Nothing]
             a.month . to_vector . should_equal [12, 2, 1, Nothing]
             a.day . to_vector . should_equal [31, 29, 1, Nothing]
-            [a.year, a.month, a.day].each c->
+            a.day_of_year . to_vector . should_equal [366, 60, 1, Nothing]
+            a.day_of_week . to_vector . should_equal [4, 4, 1, Nothing]
```

Comment:
Ideally would be good to have a test covering Sunday as well to ensure it works as advertised in all backends (as I can easily imagine some SQL backend doing it wrong by default)